examples/tabs/js/actions/WeatherActions.js

```diff
@@ -0,0 +1,27 @@
+var $ = require('jquery');
```

Comment:
if you're gonna use the ajax functionalities of jquery, you better use https://github.com/github/fetch instead